add-machine-ips.sh

```diff
@@ -13,5 +13,6 @@ for node in $(oc --config ocp/auth/kubeconfig get nodes -o template --template='
     if [[ "$machine_name" == *"worker"* ]]; then
         machine_name=$(oc --config ocp/auth/kubeconfig get machines -n openshift-machine-api | grep $node_name | cut -f1 -d' ')
     fi
+    sleep 3
```

Comment:
The error msg you posted would mean that `oc proxy` isn't up yet, and you have the sleep from before that gets run.  It happens inside link-machine-and-node.sh

Reply:
I'd move your change into that script, but also make it do a programmatic check if the proxy is ready vs just a sleep

Reply:
I agree with @russellb , the logic should be inside link-machine-and-node.sh
For example you could use curl in an until loop